src/main/java/frc/robot/Commands/SetLiftHeight.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.PIDOutput;
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+public class SetLiftHeight extends Command {
+  private double height;
+  private PIDController pidController;
+  private PIDOutput pidOutput;
+
+  public SetLiftHeight(double height) {
+    requires(Robot.lift);
+    this.height = height;
+
+  }
+
+  // Called just before this Command runs the first time
+  @Override
+  protected void initialize() {
+    this.pidOutput = new PIDOutput() {
+      public void pidWrite(double output) {
+        Robot.lift.setMotorSpeed(output);
+      }
+    };
+    this.pidController = new PIDController(0.2, 0, 0, Robot.lift.getPotentoimeter(), this.pidOutput, 0.05);
+    pidController.setAbsoluteTolerance(1);
+    pidController.setSetpoint(height);
+    pidController.setOutputRange(-1, 1);
+    pidController.enable();
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+  }
+
+  // Make this return true when this Command no longer needs to run execute()
+  @Override
+  protected boolean isFinished() {
+    return pidController.onTarget();
```

Comment:
Check out the isFinished method in this command from spikesLib:
https://github.com/Spikes-2212-Programming-Guild/SpikesLib/blob/master/src/com/spikes2212/genericsubsystems/basicSubsystem/commands/MoveBasicSubsystemWithPID.java